### DISCUSSION TITLE
AutoPreviewImageUrl slot and translated pages?

### DISCUSSION THREAD

**Discussion by brianpmccullough** [microsoft-search/pnp-modern-search#1910] [Q&A] [answered]
Hello, I am finding that the AutoPreviewImageUrl slot for some SPO translated pages (.aspx) is not valid (i.e. the generated URL doesn't actually contain any thumbnail images.  For example, if the url is:

/sites/{guid},{guid}/lists/{guid}/items/{guid}/driveItem/thumbnails/ would yield an empty array and as a result the thumbnail for the page displays as an error image.

The fix is rather trivial - to go into the translated page in edit mode, open the page details, save, and republish.

Anyone else experiencing this?

**Comment by wobba** [comment]
I would log this with Microsoft as you likely see the same with OOB experiences then.

**Reply by brianpmccullough** [reply]
Turns out there was a Power Automate running that was null-ing out the Banner Image inadvertently.  See: https://github.com/pnp/pnpjs/issues/1847

**Comment by brianpmccullough** [comment, marked as answer]
Turns out there was a Power Automate running that was null-ing out the Banner Image inadvertently. See: https://github.com/pnp/pnpjs/issues/1847
